GrassPokemon.java

```diff
@@ -0,0 +1,58 @@
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class GrassPokemon extends Pokemon {
+    private final Map<String, Integer> attacks;
+    public GrassPokemon(String name, int level, String... pokemonType) {
+        super(name, level, pokemonType);
+        attacks = new HashMap<>();
+        initializeAttacks();
+    }
+
+    private void initializeAttacks() {
+        attacks.put("leafBlade", 20);
+        attacks.put("leafStorm", 30);
+        attacks.put("bulletSeed", 15);
+        attacks.put("solarBeam", 25);
+    }
+
+    public void eats(String foodName) {
```

Comment:
Dit is overbodig. Als je de "eats" methode hier niet overschrijft, maak je al gebruik van de super. Hier over schrijven en enkel de super aanroepen, voegt niks toe.